modules/solid_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -53,7 +54,7 @@ CrackFrontDefinition::CrackFrontDefinition(const InputParameters & parameters) :
     _aux(_fe_problem.getAuxiliarySystem()),
     _mesh(_subproblem.mesh()),
     _treat_as_2d(getParam<bool>("2d")),
-    _closed_loop(false),
+    _closed_loop(getParam<bool>("closed_loop")),
```

Comment:
You should error out if the user sets "closed_loop" when they are not using crack front points.  Use params.isParamSetByUser() for that.